src/AggieEnterpriseApi/AggieEnterpriseApi.csproj

```diff
@@ -7,7 +7,8 @@
     <Company>UC Davis</Company>
     <Description>Aggie Enterprise API</Description>
     <TargetFramework>netstandard2.0</TargetFramework>
-	<LangVersion>latest</LangVersion>
+	  <LangVersion>latest</LangVersion>
+    <StrawberryShake_State>$(MSBuildProjectExtensionsPath)berry</StrawberryShake_State>

```

Comment:
Why is this added?

Reply:
Stores berry assets in `obj/berry/` instead of the default location, which is something like `C:\Program Files\dotnet\obj\berry`

Reply:
It's a better/more portable solution than requiring write access to a berry folder under program files.